build.fsx

```diff
@@ -115,15 +114,15 @@ Target.create "NuGet" (fun _ ->
 Target.create "GenerateDocs" (fun _ ->
     Shell.cleanDir ".fsdocs"
     Shell.cleanDir ".packages"
-    DotNet.exec id "tool" "uninstall --local FSharp.Formatting.CommandTool" |> ignore
-    // Use a local package store to avoid reuse of previous builds of the package with the same version
-    try 
-      Environment.setEnvironVar "NUGET_PACKAGES" (__SOURCE_DIRECTORY__ + "/.packages")
-      DotNet.exec id "tool" ("install --local --no-cache --version " + release.NugetVersion + " --add-source " + artifactsDir + " FSharp.Formatting.CommandTool")  |> ignore
-    finally
-      Environment.setEnvironVar "NUGET_PACKAGES" ""
-    DotNet.exec id "fsdocs" "build --strict --clean --property Configuration=Release" |> ignore
-    DotNet.exec id "tool" "uninstall --local FSharp.Formatting.CommandTool" |> ignore
+    // Τhe tool has been uninstalled when the
```

Comment:
Using `--tool-path` in `dotnet tool install` is more reliable and not affected by any global installation of the command tool.